src/utils/weatherDataFormatUtils.ts

```diff
@@ -76,3 +77,7 @@ export const buildExportedWeatherDataObject = (elem: WeatherData) => {
         assetId: elem.weather_condition_icon,
     };
 };
+
+export const getReversedCoordinates = (coordinates: [number, number]): [number, number] => {
+    return [coordinates[1], coordinates[0]];
```

Comment:
```suggestion
    return coordinates.reverse();
```

Reply:
ahh I cannot do this. It's annoying me really, but it seems reverse() returns a number[], but the prop needs to be a tuple [number,number]. It was the only thing that came to my mind as a solution. If you think it's odd we can have a think about this, open to any suggestions

Reply:
Let's keep it like you did. A bit annoying but it should work for now.